specification/maps/data-plane/Weather/preview/1.0/weather.json

```diff
@@ -555,6 +698,203 @@
           }
         }
       }
+    },
+    "/weather/tropical/storms/active/{format}": {
+      "get": {
+        "description": "**Get Active Storms**\n\n**Applies to**: Gen 1 (S0 and S1) and Gen 2 pricing tiers.\n\nGet all government-issued active tropical storms. Information about the tropical storms includes, government ID, basin ID, year of origin, name and if it is subtropical.",
+        "operationId": "Weather_GetActiveStorms",
+        "x-ms-examples": {
+          "GetActiveStorms": {
+            "$ref": "./examples/GetActiveStorms.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "../../../Common/preview/1.0/common.json#/parameters/ClientId"
+          },
+          {
+            "$ref": "#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "../../../Common/preview/1.0/common.json#/parameters/JsonFormat"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/ActiveStormResponse"
+            }
+          },
+          "default": {
+            "$ref": "../../../Common/preview/1.0/common.json#/responses/default"
+          }
+        }
+      }
+    },
+    "/weather/tropical/storms/{format}": {
+      "get": {
+        "description": "**Get Storm Search**\n\n**Applies to**: Gen 1 (S0 and S1) and Gen 2 pricing tiers.\n\nSearch government-issued tropical storms by year, basin ID, and government ID. Information about the tropical storms includes, government ID, basin ID, status, year, name and if it is subtropical.",
+        "operationId": "Weather_GetStormSearch",
+        "x-ms-examples": {
+          "Get Storm Search": {
```

Comment:
Can we change this to 'Get Tropical Storm Search'? I think the name should be more explicit, especially when it's listed with all the other Weather Services once in production, we want users to know it's tropical storms not regular storms.

Reply:
Updated